torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -458,7 +468,6 @@ void ProfilingGraphExecutorImpl::runProfilingOptimizations(
       // graph is essentially in a torch.no_grad context.
       UpdateDifferentiableGraphRequiresGrad(gradient.f, false);
       GRAPH_DEBUG("After UpdateDifferentiableGraphRequiresGrad ", *gradient.f);
-      runDiffGraphPasses(gradient.f);
```

Comment:
Oh man, I'm missed this two times I was reading this code!

Reply:
I missed the corresponding part in legacy executor and it took me a while to root cause the segfault in CI previously. But there's really no good solution there for legacy executor, hence the branching in `DifferentiableGraphOp::operator()`. 😢